subprojects/docs/src/docs/release/notes.md

```diff
@@ -102,6 +104,8 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 }
 ```
 
+This was done to ensure that version catalogs are accessible during the configuration phase of a multi-module project, avoiding cumbersome workarounds.
```

Comment:
@eskatos, is this an accurate statement?

Reply:
I want to make sure release notes don't just address the **what** but also the **why**. Why did we add this? How does it benefit our users?